labman/gui/templates/parse_quantification.html

```diff
@@ -7,11 +7,88 @@
 
```

Comment:
it seems like there is a fair bit of overlap with `library_pooling.html`, should the scripting logic be placed into a common library?

Reply:
A lot of the pages use this structure, but they're all slightly different. Although it may be possible to locate the scripting logic in a common library, the modifications that need to be done and the generalization of the code is complex enough that will delay the development too much.